Code of conduct draft (#28)
research-fosdem/research-fosdem.github.io: CODE_OF_CONDUCT.md

- Sustained disruption of speakers or events (verbally or physically)
- Publishing others’ private information, such as a physical or email address, without their explicit permission
- Advocating or encouraging any of the above behaviors

For more information and examples about what constitutes harassment, please refer to OpenCon’s Code of Conduct in Brief and the [Gathering for Open Source Hardware’s](https://openhardware.science/gosh-2017/gosh-code-of-conduct/) examples of behaviour.

Conference participants violating these rules may be asked to leave or removed from the Open Research devroom (virtual or in-person) at the discretion of the organisers. 

# Enforcement 
Participants asked to stop any harassing behavior are expected to comply immediately.

If a participant engages in harassing behavior, the conference organisers may take any action they deem appropriate, including warning the offender or expulsion from the online or in-person conference room. 

The organising team may take action to redress anything designed to disrupt, or with the clear impact of disrupting, the event or making the environment hostile for any participants.

> # Reporting

Author: We mentioned something here about having two points of contact - @diegantobass and @yochannah? Do we want to have your information here directly, or something less specific in the written form?

Maybe adding more information: "Alternatively, at both the in-person and online events, participants can speak to any of the organisers directly. Two devroom organisers will be designated as points of contact for any participants."

If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of the devroom organising team immediately. 

> Instances of abusive, harassing, or otherwise unacceptable behavior may also be reported to the organisers responsible for enforcement at [anonymous Google form?]

Author: ```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior may also be reported to the organisers responsible for enforcement at [framaform linked here]. 
```

# Recusal
Conference organisers and any others responsible for enacting or overseeing this policy will recuse themselves if they have a significant conflict of interest, such as being a specific target of harassment, an alleged harasser, or having a close personal or professional relationship with a target or alleged harasser.

# 5 Acknowledgement
This code of conduct draws upon the [Contributor Covenant](https://www.contributor-covenant.org/) and the [Conference Code of Conduct](https://confcodeofconduct.com/) (CC BY 3.0). 

It also takes inspiration from specific Codes of Conduct that were written for aligned communities and conferences events. This includes the following: 

Communities and projects: 
- The Turing Way: https://github.com/the-turing-way/the-turing-way/blob/main/CODE_OF_CONDUCT.md
- Open Life Science: https://github.com/open-life-science/policies-procedures-and-docs/blob/main/CODE_OF_CONDUCT.md
- Frictionless Data: https://frictionlessdata.io/work-with-us/code-of-conduct/

Events and conferences

> - csv,conf: (https://csvconf.com/coc/)

Author: ```suggestion
- [csv,conf](https://csvconf.com/coc/)
```

- Mozilla Festival: 

> - eLife Sprint Code of Conduct

Author: ```suggestion
- eLife Sprint Code of Conduct: https://sprint.elifesciences.org/index.html
```

> - PyCon Code of Conduct 

Author: ```suggestion
- PyCon Code of Conduct: https://2023.pyconuk.org/code-of-conduct/
```
